lib/cib/cib_utils.c

```diff
@@ -734,12 +734,20 @@ cib__signon_query(cib_t **cib, xmlNode **cib_object)
         return ENOMEM;
     }
 
+    crm_trace("Connecting to the CIB");
```

Comment:
This is fine, but FYI signon() will log a message too

Reply:
It's a little bit less specific and less timely for cib_file and cib_remote client signon. I think the signon logging is fine though.